src/_internal/Rpc.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.13 <0.9.0;
+
+import {vulcan} from "./Vulcan.sol";
+import {forksUnsafe, Fork} from "./Forks.sol";
+import {JsonObject} from "./Json.sol";
+
+library rpc {
+    /// @dev Calls an JSON-RPC method on a specific RPC endpoint. If there was a previous active fork it will return back to that one once the method is called.
+    /// @param url The url of the RPC endpoint to use
+    /// @param method The JSON-RPC method to call
+    /// @param params The method params as a JSON string
+    function call(string memory url, string memory method, string memory params) internal returns (bytes memory data) {
```

Comment:
maybe it is better to implement all the logic in the `call` function that receives a JsonObject, that way the params are validated?

Reply:
That's a good idea. Should be solved now